harness/determined/experimental/client.py

```diff
@@ -289,10 +289,13 @@ def logout() -> None:
 
 
 @_require_singleton
-def list_users() -> List[User]:
+def list_users(active_only: bool) -> List[User]:
     """Get a list of all Users."""
     assert _determined is not None
-    return _determined.list_users()
+    if active_only:
+        return _determined.list_users(active=True)
+    else:
+        return _determined.list_users()
```

Comment:
```suggestion
    return _determined.list_users(active=active)
```

Reply:
active=False != all users

Reply:
oh i see what you mean, this plus the comment above. gotcha.

Reply:
writing a unit test for it is preferred.  There's not a lot of examples of that just yet, unfortunately.